include/volt.h

```diff
@@ -9,9 +9,39 @@
 #include <Arduino.h>
 #include <farmSensor.h>
 
+class volt : public farmSensor {
+private:
+    int m_RH; // voltage divider, higher resistor value [ohms]
+    int m_RL; // voltage divider, lower resistor value [ohms]
+protected:
+    float readRaw() override; 
+public:
+    static const int adc_res = 4095; // ESP32 ADC resolution is 12 bits
```

Comment:
I think this should be a private member.
I do not see a reason why we need to access this from `main.cpp`